src/components/10-atoms/input-file/index.js

```diff
@@ -0,0 +1,97 @@
+import { html } from 'lit-element';
+import { classMap } from 'lit-html/directives/class-map';
+import { unsafeHTML } from 'lit-html/directives/unsafe-html';
+/* eslint-disable import/no-extraneous-dependencies */
+import '@axa-ch/icon';
+import defineOnce from '../../../utils/define-once';
+import NoShadowDOM from '../../../utils/no-shadow';
+import styles from './index.scss';
+
+const TYPE = 'file';
+class AXAInputFile extends NoShadowDOM {
+  static get tagName() {
+    return 'axa-input-file';
+  }
+
+  static get styles() {
+    return styles;
+  }
+
+  static get properties() {
+    return {
+      // secondary, red,  inverted, inverted-green, inverted-dark-blue
+      variant: { type: String },
+      icon: { type: String },
+      id: { type: String },
+      large: { type: Boolean },
+      motionOff: { type: Boolean },
+      disabled: { type: Boolean, reflect: true },
+      accept: { type: String },
+      capture: { type: Boolean },
+      multiple: { type: Boolean },
+    };
+  }
+
+  constructor() {
+    super();
+    this.variant = '';
+    this.icon = '';
+    this.refId =
+      this.id ||
+      `input-file-${Math.random()
+        .toString(36)
+        .slice(2)}`;
+    this.large = false;
+    this.motionOff = false;
+    this.disabled = false;
+    this.accept = 'image/jpg, image/jpeg, application/pdf, image/png';
```

Comment:
Do we really want to restrict the file-list by default?

Reply:
yes, cause everything open leaves space for zip which it get forgotten can have limitation of WAF or ESG

Reply:
User can always override if they wan to

Reply:
Leave space open for zip? What does that mean?

Is it bad if a zip file gets uploaded?

Reply:
without security on the backend, yes is really bad

Reply:
But security belongs to the backend. The backend has to make sure, that it is secured against attacks. Doing the security in the frontend while leaving the backend-endpoints naked is a flaw. But doesn't matter for this PR I guess. Just saying that we should never do security work because a backend might potentially not handle it. Then the backend needs an update. One could also just enable the zips in the frontend with the developer tools and still be able to update it, so that's not a proper security measure.